public/locales/zh/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Nix 生态链的一个替换方案",
+
+	"header.community": "社区",
+
+	"goals.title": "目标",
+	"goals.independent.title": "独立",
+	"goals.independent": "我们将为 Nix、NixPkgs 和 NixOS 的存储库创建分支，并维护及分支出的代码，也将考虑对 Nix Darwin 和 Home Manager 等项目提供支持。",
+	"goals.gov.title": "治理",
+	"goals.gov": "本项目将用一个民主治理模式来运行，及拥有选举出的职位。一个指导委员会将提供主要领导，由其他委员会负责处理其他具体后勤任务。 实际功能及单个项目将拥有于特殊兴趣小组（“Special Interest Groups“，又称 “SIGs”）。这些特殊兴趣小组将与工作组（“Working Groups”）合作，一起达成目标。每个委员会将清楚定义各自的目标、计划、要求，以及管理流程。",
+	"goals.stabilization.title": "稳定性",
+	"goals.stabilization": "拥有良好用户体验的功能例如 Nix v3 的命令行工具将被默认设置为启动状态。由于Nix Flakes 的市面用途太广泛，它不将被轻易改动或甚至去除，然而它将以当下的状态下以 v0 版本被稳定化，今后的任何其他改进将有一个特殊兴趣小组来负责。",
+	"goals.infra.title": "设施",
+	"goals.infra": "我们将构建及维护生态系统的健康所需要的工具、平台，及基础设施，包括持续集成（“Continuous Integration”）和二进制缓存（“Binary Caching”）。",
+	"goals.education.title": "教育与科普",
+	"goals.education": "统一的科普文档、教程、指南和示例将为新手提供更好的入门体验，同时为专家提供更快的迭代周期。",
+
+	"roadmap.title": "阶段路线图",
+	"roadmap.soft_fork.title": "软分支",
+	"roadmap.soft_fork.1": "由于该项目现处于初步阶段，该阶段将采纳临时性质的管理结构。因此，项目规定的价值观、目标及路线图有助于让所有贡献者保持一致并安全地知道大家所承诺的工作不会改变。",
+	"roadmap.soft_fork.2": "在此阶段，第一步为 Nix 及 NixPkgs 做分支。代码贡献者会被分配关于程序包管理及从上游抓取新的改进的任务。一旦有足够多的贡献者加入项目，我们打算从这个阶段继续，以便能够过渡到硬分支。",
+	"roadmap.hard_fork.title": "硬分支",
+	"roadmap.hard_fork.1": "与软分支相似，会采取临时的管理模式，但委员会、特殊兴趣小组及工作小组将足部构建起来。由于每个阶段都有必要的引导步骤，只有在达到“对齐”阶段后才开始选举。",
+	"roadmap.hard_fork.2": "此时，我们将有足够的贡献者来一起维持（或部分的）NixPkgs 及 Nix。我们将足部允许从上游产生代码分支，此时从上游的代码结构的改变也能被采纳。",
+	"roadmap.hard_fork.3": "我们将推出最初步代码包版本的发布，从`nix` 搬到 `aux` 流程将局部发展，这样来减少困惑及报错等问题。",
+	"roadmap.organization.title": "组织",
+	"roadmap.organization.1": "我们将把部分的代码包的代码库提取出，由特殊兴趣小组来更方便的管理各自的流程。例如，Python 的代码包将被转出成独立的代码库，由Python 兴趣小组来管理。",
+	"roadmap.organization.2": "Flakes 将以当下的状态以 v0 版本来被稳定化。虽然这不一定最佳选择，但由于它的市面用途太广泛，不容易被轻易改动或去除，导致生态链的损坏。今后的改进将会专门被 Flakes 兴趣小组来领导。",
+	"roadmap.organization.3": "`aux` 的命令行工具将采纳更人性化管理代码包和系统的改进。我们也将增加格外的命令，如 `aux system switch` 及 `aux system build` 来把系统就职及管理的任务变得更方便。",
+	"roadmap.infra.title": "设施与平台",
+	"roadmap.infra.1": "现在该项目已与上游显着偏离，我们需要提供持续集成和二进制缓存服务。现有的治理结构将用于管理这些技术的采用。",
+	"roadmap.infra.2": "帮助文档的相关材料将作为持续集成流程的一部分包含在内，并将配置持续部署以使其保持最新状态，从而使贡献者们能够更轻松的创建科普材料。",
+	"roadmap.infra.3": "我们还会创建软件包和选项的搜索的基础设施。",
+	"roadmap.alignment.title": "对齐阶段",
+	"roadmap.alignment.1": "此时，我们做到了。我们创建了 Nix 生态系统的一个可持续的、独立的分支。初步工作已经完成，我们将开始治理的首次轮换过程，进行选举及关于如何运行操作的轮换科普。",
```

Comment:
IMO this sounds more like "We did" rather than "We did it" but my attempts to rectify this had ended up becoming too verbose and added extraneous, unintended connotations.